### DISCUSSION TITLE
no an issue but a question

### DISCUSSION THREAD

**Discussion by costascontis** [PiotrMachowski/Xiaomi-cloud-tokens-extractor#96] [Q&A]
Smartmi Link app is similar to Mihome,is there a  way to extract tokens from there?

**Comment by PiotrMachowski** [comment]
I have never used it. Is it possible to login to it using Xiaomi Cloud credentials?

**Reply by costascontis** [reply]
no i dont think so...The thing is that i got fooled and bought a [[smartmi-circulating-fan](url)](https://eu.smartmiglobal.com/pages/smartmi-circulating-fan) that is google/Alexa/Homekit supported but not MIhome!!!I use Openhab/mqtt/zigbee setup and i dont have a way to intergrate this fan.The fan is wondrerfull small ,strong full with features but its not mihome supported.They have an app SmartLink that the fan connects using bt and wifi.....Do you have an idea what to do ?
